docassemble/ALDashboard/data/questions/install_fonts.yml

```diff
@@ -0,0 +1,88 @@
+---
+include:
+  - nav.yml
+---
+metadata:
+  required privileges:
+    - admin
+---
+modules:
+  - .aldashboard
+---
+mandatory: True
+code: |
+  the_font_files
+  if install_fonts_task.ready():
+    ending_screen
+  else:
+    waiting_screen
+---
+id: waiting screen
+event: waiting_screen
+question: |
+  Installing fonts
+subquestion: |
+  <div class="spinner-container d-flex justify-content-center">
+  <div class="spinner-border" role="status">
+    <span class="visually-hidden">Installing fonts...</span>
+  </div>
+  </div>
+
+  This screen will reload every
+  ten seconds until the installation is complete.
+reload: True    
+---
+id: ending screen
+event: ending_screen
+question: |
+  % if install_fonts_task.failed():
+  Installation failed
+  % else:
+  Installation successful
+  % endif
+subquestion: |
+  % if install_fonts_task.failed():
+  ${ install_fonts_task.ready() }
+  % endif
+---
+code: |
+  install_fonts_task = background_action("install_fonts_event")
+---
+event: install_fonts_event
+code: |
+  install_fonts(the_font_files)
+  background_response()
+---
+id: install fonts
+question: |
+  Upload the fonts you want to install
+subquestion: |
+  You can upload multiple fonts at once. The fonts will be installed on the server and will be available for use in your interviews.
+
+  ${ action_button_html(url_action("list_fonts_task"), label="View installed fonts", id_tag="view_fonts") }
+
+  [TARGET displayed_fonts]
+
+fields:
+  - Font files: the_font_files
+    datatype: files
+    #accept: |
+    #  "application/font-sfnt, .TTF, .ttf, font/*"
```

Comment:
Was this not working for some reason? Was going to suggest it, only seeing the files you're allowed to upload really helps.

Reply:
It didn't work. When it was enabled, it was blocking any upload of a TTF straight from c:\windows\fonts